modules/dropdown/js/dropdown_directive.js

```diff
@@ -158,9 +157,12 @@
             }
             enableBodyScroll = undefined;
 
+            var dropdownToggleElement;
             if (lxDropdown.hasToggle)
             {
-                dropdownToggle
+                dropdownToggleElement = (angular.isString(dropdownToggle)) ? angular.element(dropdownToggle) : dropdownToggle;
```

Comment:
ok following our discussion, I see the use case you need this for. Basically perf will be slightly worse if we use a string but we may need it in case the original referenced element is removed from the DOM.